pkg/client/group_mgmt_client.go

```diff
@@ -219,14 +220,45 @@ func (client *GroupMgmtClient) List(path string) (interface{}, error) {
 
 // ListFromParams :
 func (client *GroupMgmtClient) ListFromParams(path string, params *param.GetParams) (interface{}, error) {
-	listResp, err := client.listGetOrPost(path, params)
+
+	// pagination, set the start row and end row if the pagination is enabled.
+	// TotalRows is nil if this is first pagination request.
```

Comment:
I'm not sure the "move to next page" logic belongs here.  The caller to this function should provide the parameters for what it wants, not what was returned from the previous call.

Maybe what we need is a function like this:

func (page *GetPage) GetNextPage() GetPage {
  return Page{
    StartRow: page.EndRow
    EndRow: page.EndRow - pageStartRow + pageEndRow
  }
}

or this:

func (page *GetPage) Advance() {
  pageSize := page.EndRow - page.StartRow
  page.StartRow = page.EndRow
  page.EndRow = page.EndRow + pageSize
}